.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,7 @@
-**Description:**
```

Comment:
Is the though here that everyone know to add a useful description?

Reply:
For a single-commit PR, GitHub automatically suggests the commit's body here. I think if people aren't putting in useful commit bodies, they're not going to make useful PR descriptions anyhow. (I don't think I've _ever_ seen someone use this prompt, in practice!)